### DISCUSSION TITLE
Changing Title Separator

### DISCUSSION THREAD

**Discussion by seeergiofdez** [facebook/docusaurus#10277] [Q&A] [answered]
Hi!
I've been searching in the documentation for a way or method to change the title separator. It's currently using "|" separator, between the doc title and the website name. Is there any way to change it to other character (like "-", or "—"?

Thanks!

**Comment by slorber** [comment, marked as answer]
https://docusaurus.io/search?q=Title

https://docusaurus.io/docs/api/docusaurus-config#titleDelimiter

**Reply by seeergiofdez** [reply]
Thank you very much! It seems that I didn't read the whole documentation very well...
